### ISSUE TITLE
Support for Specific Frontend Frameworks like Vue/Element?

### ISSUE THREAD

**Issue by jianzhis** [abi/screenshot-to-code#106]
I'm wondering if there is support for specific frontend frameworks, such as Vue/Element. Having compatibility with these frameworks would significantly reduce development time for me, as my current company and role mandate the use of a designated frontend framework, with little usage of Tailwind CSS. Any insights or guidance on this matter would be greatly apprecia

**Comment by abi** [comment]
Yes that is the goal. We have a couple of prototypes in the works. 

The current set of frameworks/libraries we intend to support are:

- React
- Vue
- Ionic framework
- Pure CSS instead of tailwind
- Element?

Let us know what else you are interested.

**Comment by NameQuChen** [comment]
ant 
naiveui

**Comment by chaos-zhu** [comment]
> Yes that is the goal. We have a couple of prototypes in the works.
> 
> The current set of frameworks/libraries we intend to support are:
> 
> * React
> * Vue
> * Ionic framework
> * Pure CSS instead of tailwind
> * Element?
> 
> Let us know what else you are interested.

Element --> Element-ui(https://element.eleme.io/)
